log/gps/gps_destinator_trk.ksy

```diff
@@ -0,0 +1,82 @@
+meta:
+  id: gps_destinator_trk
```

Comment:
Unless "trk" is a file extension or a common name for the format, I would spell it out completely.

```suggestion
  id: gps_destinator_track
```